drivers/classic/src/mtclim_wrapper.c

```diff
@@ -188,7 +195,6 @@ mtclim_init(int               have_dewpt,
         ctrl->insw = 0;
     }
     ctrl->outhum = 1;           /* output vapor pressure */
-    ctrl->inyear = 0;
```

Comment:
Yes it is internal to mtclim but it is never used, it was allocated and initialized but is not involved anywhere else in the code.